.github/workflows/ci.yml

```diff
@@ -167,3 +167,30 @@ jobs:
 
       - name: Build Docker image
         run: ./run-tests.sh --check-docker-build
+
+  release-docker:
+    runs-on: ubuntu-20.04
+    if: >
+      vars.RELEASE_DOCKER == 'true' &&
+      github.event_name == 'push' &&
+      startsWith(github.ref, 'refs/tags/')
+    needs:
+      - lint-shellcheck
```

Comment:
Cosmetics: let's capitalise the "name" as in "Release Docker image" for consistency with the other rules we have, and let's order the "needs" alphabetically.